README.md

```diff
@@ -126,18 +123,7 @@ module.exports = function(config) {
 > Type: `Boolean`
 > Default: `true`
 
-If we should prefer running headless browsers. This will change the defaults for `travisLaunchers` as well as automatic browser detection. Make sure to handle this in [`browsers`](###browsers)
-
-### `detectBrowsers`
```

Comment:
why remove detectBrowsers?

Reply:
We only detect browsers when browsers are not explicitly defined. It seems rather pointless to skip detect browsers when no browsers are set as *nothing* will run, causing an error.

Reply:
Ah, makes sense. Probably worth documenting that.
Plus, since this change is already a breaking change there's no issue in removing an option. Though, should this be a breaking change? I guess it probably doesn't matter that much since the functionality is the same, and we're not looking to add other functionality that could be useful to travis/TC items directly yet.

Reply:
I think for docs maybe something like `detect browsers automatically if no browsers were configured` under the `browsers` option?

Reply:
yeah that makes sense I will add that under the browsers option. I think this pull request was going to be a breaking change anyway just to be safe.